Rakefile.rb

```diff
@@ -28,7 +28,7 @@
     header="<% name=\"#{ENV['INTEGRATION']}\" %> \n <% version=\"#{ENV['VERSION']}\" %> \n <% build_iteration=\"#{ENV['BUILD_ITERATION']}\" %>"
     sh "(echo '#{header}' && cat #{PROJECT_DIR}/resources/datadog-integrations/project.rb.erb) | erb > #{PROJECT_DIR}/config/projects/dd-check-#{ENV['INTEGRATION']}.rb"
 
-    header="<% name=\"#{ENV['INTEGRATION']}\" %> \n <% core_integrations=\"#{CORE_INTEGRATIONS_DIR}\" %>"
+    header="<% name=\"#{ENV['INTEGRATION']}\" %> \n <% core_integrations=\"#{CORE_INTEGRATIONS_DIR}\" %> \n <% PROJECT_DIR=\"#{PROJECT_DIR}\" %>"
     sh "(echo '#{header}' && cat #{PROJECT_DIR}/resources/datadog-integrations/software.rb.erb) | erb > #{PROJECT_DIR}/config/software/dd-check-#{ENV['INTEGRATION']}-software.rb"
```

Comment:
not the worst thing, but I think this could use a helper function, it's kind of hacky :) so might as well add a note to say that erb does not support setting template variables on the command line.